x-pack/legacy/plugins/uptime/public/lib/adapters/source_settings/index.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export { getSourceSettings } from './get_source_settings';
```

Comment:
Would you mind updating CODEOWNER with uptime app ?

Reply:
can do